src/SignatureParametersParser.php

```diff
@@ -100,8 +100,8 @@ private function validate($result)
     private function validateAllKeysArePresent($result)
     {
         // Regexp in pair() ensures no unwanted keys exist.
-        // Ensure that all wanted keys exist.
-        $wanted = ['keyId', 'algorithm', 'headers', 'signature'];
+        // Ensure that all mandatory keys exist.
+        $wanted = ['keyId', 'algorithm', 'signature'];
```

Comment:
this change needs a corresponding test

Reply:
Mandatory signature fields absent test:
https://github.com/liamdennehy/http-signatures-php/blob/e8d9b91904f43f80af78bca6cacbc72208669a06/tests/VerifierTest.php#L122
Missing signature fields test:
https://github.com/liamdennehy/http-signatures-php/blob/e8d9b91904f43f80af78bca6cacbc72208669a06/tests/VerifierTest.php#L128
What else are we looking for?

Reply:
Closing...